tests/test_match.py

```diff
@@ -395,7 +395,7 @@ def test_known_indel(self, conn):
         assert 'EGFR mutation' in names
         assert 'EGFR copy variant' not in names
 
-    def test_movel_specific_matches_general(self, conn):
+    def test_novel_specific_matches_general(self, conn):
```

Comment:
It would probably be good idea to have the tests on the `compare_positional_variant` instead with the 2 records you gave in your comment above (strip out the unecessary fields for simplicity)